src/SketchPad.jsx

```diff
@@ -55,7 +55,7 @@ export default class SketchPad extends Component {
   }
 
   componentDidMount() {
-    this.canvas = findDOMNode(this);
+    this.canvas = findDOMNode(this.canvas);
```

Comment:
can you please rename it to `canvasRef`? And don't forget to update it in `render` method
```
this.canvas = findDOMNode(this.canvasRef);
```
I'm not happy with `ref` but there's no other way so I hope it's better to add it to variable name.